### ISSUE TITLE
Biorefinery & Transmat failed to parse

### ISSUE THREAD

**Issue by antool** [agroportal/project-management#128]
Error from parsing log file (Biorefinery): Illegal rdf:nodeID value '_:genid259'
there is an equivalent error for Transmat.
This error seems to have been identified in the NCBO BioPortal: see 
- [https://github.com/ncbo/bioportal-project/issues/32#event-1226205997](https://github.com/ncbo/bioportal-project/issues/32#event-1226205997)
- [https://github.com/ncbo/bioportal-project/issues/9](https://github.com/ncbo/bioportal-project/issues/9)
- https://sourceforge.net/p/owlapi/mailman/owlapi-developer/thread/CAD2jOMO3yMeQ7Ojk7TtNdMoshqQCQ87ke37aQWJDNeqZzvCAew%40mail.gmail.com/#msg35964428
@jvendetti Did you solve this problem?

**Comment by twktheainur** [comment]
Fix in owlapi 4.3.2, already integrated in NCBO BioPortal. 
We need to merge our project with upstream, regenerate the jar and updated it in LIRMM branch of ontologies_api

**Comment by antool** [comment]
The Biorefinery (and Transmat) ontology failed to parse because it encounters what the log claims to be an 'illegal blank node identifier':
` Illegal rdf:nodeID value '_:genid259'`
The ontology that was originally submitted didn’t contain any blank node. The blank nodes were added only after we parsed the ontology with the OWL API 4.3.1. integrated in AgroPortal.
The problem has been fixed in the next version of OWL API (release 4.3.2).
We just had to update the OWL API in AgroPortal as explained before by @twktheainur.
